rebasehelper/tests/plugins/test_spec_hooks.py

```diff
@@ -83,6 +83,10 @@ def test_replace_old_version_spec_hook(self, spec_object, replace_with_macro):
             expected_result = 'https://test.com/#/1.1/%{name}-hardcoded-version-1.1.0.tar.gz'
         assert test_source[0].split()[1] == expected_result
 
+        # Check if dependency tags are ignored
+        test_recommends = new_spec.get_raw_tag_value('Recommends')
+        assert test_recommends == 'test > 1.0.2'
```

Comment:
Just a nitpick, but this could be oneliner.

Reply:
Good catch